pkgs/os-specific/darwin/apple-source-releases/Libsystem/system_kernel_symbols

```diff
@@ -790,6 +790,8 @@ _removexattr
 _rename
 _rename_ext
 _renameat
+_renamex_np
+_renameatx_np
```

Comment:
Are these available on 10.12 and earlier?

Reply:
It looks like they were introduced in xnu-3789.70.16 which corresponds to 10.12. gnulib uses it since this commit:

https://github.com/coreutils/gnulib/commit/9abc08406d458ce18eb98a286aaae618a33dd231

and from updating to latest xnu:

https://github.com/apple/darwin-xnu/blob/0a798f6738bc1db01281fc08ae024145e84df927/bsd/sys/stdio.h#L47-L48

Here is the file that defines it in XNU:

http://newosxbook.com/src.jl?tree=xnu-3789.70.16&file=/libsyscall/wrappers/renamex.c

I wonder what people think about breaking support for 10.10 and 10.11? How common are they in the wild?

Reply:
Well 10.12 is pretty important because I'm pretty sure most builders are still running that. We probably don't have to keep older versions, but we should officially drop support for those then.

Reply:
I was working on automating the generation of those symbol lists as a step towards #23432
https://github.com/veprbl/libsystem_symbol_list/branches <- tool to collect lists
https://github.com/veprbl/nixpkgs/commits/libsystem_symbols <- branch intended for the PR, includes the lists and scripts to take intersection
I was never able to dedicate CPU time to bootstrap this before making a PR, but if someone would be interested in picking this up, feel free :)

Reply:
Yeah that definitely seems useful. We don't want to use the symbols from the latest libsystem just because it would break for older versions. I wonder if anyone is still running 10.12 who could run the script? Most likely there's only a few new symbols.

Reply:
@veprbl Were there any issues with this method? Might be worth putting in this PR.

Reply:
There is a 10.12.6 branch in libsystem_symbol_list that was acquired from Travis machine. It would be nice to have a volunteer to contribute a list for 10.14.

Reply:
@matthewbauer Not that I know of. In the end of the day you just get a list and the rest of the machinery stays the same. You are not able to easily edit it, but it least its content have a clear motivation. And you also recover some missing symbols that used to be skipped in manually maintained list.